scripts/fast-reboot

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash -e
 
-REBOOT_USER=$(logname)
+# reboot DBus service, which uses this script, doesn't use a login shell
+REBOOT_USER=$(logname || whoami)
```

Comment:
Minor comment - use `REBOOT_USER=$(logname 2>/dev/null || whoami)`?

If logname fails, its error message will be printed before whoami is executed. By redirecting stderr to `null`, you prevent the error message from cluttering the logs or terminal output.